vignettes/algorithm.Rmd

```diff
@@ -157,6 +157,22 @@ The parametrization of $\theta$ is similar to other non-grouped $\theta$.
 Assume that there are total number of $G$ groups, the length of $\theta$ is multiplied by $G$, and for each part, $\theta$ is parametrized in the same fashion.
 For example, for an unstructured covariance matrix, $\theta$ has $k = G * m(m+1)/2$ entries.
 
+#### Spatial covariance matrix
+
+Under spatial covariance structure, the covariance matrix is different for all subjects, if they have different visit timepoints.
+Following the notations in the previous section, for subject $i$ with total number of $m_i$ visits, we have
+
+\[
+\Sigma_{i}(m_{ij}, m_{ik}) = \sigma * f(dist(t_{m_j}, t_{m_k}))
```

Comment:
I find the notation can be simplified probably - e.g. you could write
```suggestion
\sigma_{ijk} = \sigma * f(dist(\boldsymbol{c}_{ij}, \boldsymbol{c}_{ik}))
```
and explain that this is the j, k element of the covariance matrix for patient i, and c are the corresponding coordinate vectors.